massgov/mayflower-patternlab
# DP-8825 Add expandable teaser listing for org boards

## Description

Adds a variation of the teaser listing organism to allow for an accordion layout.


## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8825)

## Steps to Test


1. Run maflower locally
2. Ensure than the existing listing table organism works and looks as it used to
3. Look at the organization boards page, and see that it matches these invision links:
    - https://invis.io/6WEP0CK7M#/295217761_orgboardclosed
    - https://invis.io/6WEP0CK7M#/295217760_orgboardopen 


## Screenshots



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* Added `@organisms/by-author/teaser-listing~as-expandable

#### @TODO


* Maybe add blocks around each item subsection?

#### Today I learned...
